R/server.R

```diff
@@ -163,10 +163,9 @@ capabilities <- function() {
   )
 }
 
-# Hacky way of getting tools from btw
+# TODO: btw now surfaces a btw_tools() function that can be passed
+# directly to `tool_as_json()` along with .acquaint_tools (#12)
 get_all_btw_tools <- function() {
```

Comment:
This can all be simplified a good bit with #12, but keeping that set of changes isolated for now.